updater/update.py

```diff
@@ -73,11 +73,11 @@ def update():
 	print("[+] Starting Update generic information")
 
 	# get generic data 
-	r = requests.get('https://www.root-me.org/fr/Communaute/Classement/')
-	regex = r'<a href=".*?>([0-9]+)</a>'
```

Comment:
check with : 

```
r'</a>'<td><a href=".*" title=".*">([0-9]+)</a></td>'
```

Reply:
I don't see where you can find the total_points value in this page to be honest.
I'm testing with Chrome and don't see anything.

I think you just catch the score of the number one in the scoreboard, don't you?

Reply:
This is a poor solution I have to admit ^^ Your solution is better 👍 PR accepted